tests/test_mdspan_size.cpp

```diff
@@ -58,9 +58,6 @@ TEST(TestMdspan, MdspanSizeReturnTypeAndPrecondition)
 
   static_assert(std::numeric_limits<std::int8_t>::max() == 127, "max int8_t != 127");
   test_mdspan_size(storage, Kokkos::extents<std::int8_t, 12, 11>{}); // 12 * 11 == 132
-
-  static_assert(std::numeric_limits<std::uint8_t>::max() == 255, "max uint8_t != 255");
-  test_mdspan_size(storage, Kokkos::extents<std::uint8_t, 16, 17>{}); // 16 * 17 == 272
```

Comment:
This isn't supposed to work according to standard, because size() returns size_type, the unsigned version of index_type

Reply:
I don't understand that change/comment.
Everything else looks good.

Reply:
We previously returned size_t so mdspan::size() would work even if it overflowed uint8_t, but mdspan size return the unsigned version of index_type: so we can't return 272

Reply:
Wanna do 16, 15 then?